config.m4

```diff
@@ -496,6 +533,7 @@ dnl }}}
 
   BSON_EXTRA_ALIGN=0
   AC_SUBST(BSON_EXTRA_ALIGN)
+  AC_SUBST(BSON_HAVE_DECIMAL128, 0)
```

Comment:
Thanks for clarifying `BSON_HAVE_DECIMAL128`. I misunderstood the purpose of the constant.

> Similarly, you'll want on your BSON\Decimal128 class to provide, when available: `getAsGmp()`, `getAsBCMath()`

Cross-referencing with existing discussion in [PHPC-640](https://jira.mongodb.org/browse/PHPC-640).